src/Module/Hooks/OrdersGridHooks.php

```diff
@@ -35,7 +37,7 @@ public function hookActionOrderGridQueryBuilderModifier(array $params)
         $searchQueryBuilder->addSelect('o.id_cart');
         $searchQueryBuilder->leftJoin(
             'o',
-            _DB_PREFIX_ . 'carrier',
+            Table::withPrefix('carrier'),
```

Comment:
ik zou dit in een aparte variabele plaatsen.

Reply:
Is er niet een `TABLE::TABLE_CARRIER` ?

Reply:
Ik heb alleen de tables die de plugin zelf toevoegt vastgelegd, dit is van prestashop zelf

Reply:
Heb er een variabele van gemaakt en de sql opgeschoond